src/current/v24.2/performance-benchmarking-with-tpcc-local.md

```diff
@@ -29,43 +29,29 @@ This page shows you how to reproduce [CockroachDB TPC-C performance benchmarking
 
 {% include {{ page.version.version }}/prod-deployment/insecure-flag.md %}
 
-1. In separate terminal windows, use the [`cockroach start`]({% link {{ page.version.version }}/cockroach-start.md %}) command to start 3 nodes:
+Follow these steps to install CockroachDB and start a three-node cluster on the local system:
+
+1. Visit [Releases]({% link releases/index.md %}?filters=windows) to download and CockroachDB for Linux. Select the architecture of the VM, either Intel or ARM. Releases are rolled out gradually, so the latest version may not yet be available.
+
+1. Extract the binary you downloaded, then optionally copy it into a location in your `PATH`. If you choose to copy it into a system directory, you may need to use `sudo`.
+
+1. Open three terminals. In each terminal, start CockroachDB. Use a different port for each node's `--listen-address` and `http-address`, and add each node's `--listen-address` to the `--join` flags for all nodes.
```

Comment:
same as above, I do not think we should be refactoring the working cluster start instructions here